domains/runtime/core-eth-relay/Cargo.toml

```diff
@@ -25,15 +25,16 @@ log = { version = "0.4.17", default-features = false }
 pallet-balances = { version = "4.0.0-dev", default-features = false, git = "https://github.com/subspace/substrate", rev = "fdb68194ab6995447610b3dbdee70559711dbd63" }
 pallet-domain-registry = { version = "0.1.0", path = "../../pallets/domain-registry", default-features = false }
 pallet-executor-registry = { version = "0.1.0", path = "../../pallets/executor-registry", default-features = false }
+pallet-feeds = { version = "0.1.0", path = "../../../crates/pallet-feeds", default-features = false }
```

Comment:
You forgot to add `"pallet-feeds/std"` to the std list, it's interesting that the compiler said nothing.

Reply:
You are right. Added it.